### ISSUE TITLE
window.ga is not a function in Mobile Safari

### ISSUE THREAD

**Issue by cspeer** [MatteoGabriele/vue-analytics#17]
Hi,

I'm running on the latest version 2.3. and I get the following error from a few iOS 10.2.1 devices with Mobile Safari 10:
`window.ga is not a function. (In 'window.ga("create",i.id,"auto",r)', 'window.ga' is undefined)`

Otherwise everything is fine.

Thanks,
Chris

**Comment by MatteoGabriele** [comment]
@cspeer I can't reproduce this error, but i can only test until 10.2. 
The version you have has not been released on XCode yet.

Could you provide an example of your setup please? Would help a lot.

Thanks

**Comment by cspeer** [comment]
You're right, it has not been released for the iOS simulator. And testing on my personal device does not bring up this issue. However, this issue pops up for some other users. 
I'll attach a screenshot of the error report in Sentry.

Here is my setup:
```javascript
'use strict';

import Vue from 'vue';
import VueAnalytics from 'vue-analytics';
import router from './router';

if(process.env.NODE_ENV !== 'development') {
  const config = require('../config/google.json')[process.env.NODE_ENV];

  Vue.use(VueAnalytics, {
    id: config.analytics,
    router
  });
}
```

Thanks for looking into this.

**Comment by MatteoGabriele** [comment]
Ok thanks for the sentry report, I'm going to have a look a the method that loads the script, maybe there's something strange going there.

Btw you can easily disable VueAnalytics just adding this properties to the debug object and you won't track or send hints to the domain associated

```js
Vue.use(VueAnalytics, {
  id: 'UA-xxxxxxx-x',
  router,
  debug: {
    enabled: false,
    trace: false,
    sendHitTask: false
  }
})
```

**Comment by cspeer** [comment]
Thanks Matteo. I appreciate your effort!

**Comment by MatteoGabriele** [comment]
@cspeer thanks to you for the issue.

**Comment by MatteoGabriele** [comment]
Due to the new update on the OS, I've added as a low priority but I want to address it asap as always :)

**Comment by MatteoGabriele** [comment]
I've just released a v3.0.0 which also have your fix.

The `ga` object was called before the script was added in the dom:  that's why `window.ga` was resulting undefined.

Let me know of future issues.

**Comment by cspeer** [comment]
Thanks Matteo! That's awesome!

**Comment by MatteoGabriele** [comment]
man this safari browser! i hate it! ahhahahah
gonna have a look! :)

**Comment by MatteoGabriele** [comment]
Yeah, indeed!

**Comment by zombor** [comment]
I got this same error today, btw!

**Comment by MatteoGabriele** [comment]
published v3.0.1 :tada:

now I start polling until `ga` is defined. I don't like it but in this cases is necessary.
Let me know guys if you have other issue!

thanks!

**Comment by cspeer** [comment]
Thanks again Matteo
